about.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script src="https://kit.fontawesome.com/870e4168d3.js" crossorigin="anonymous"></script>
+    <script
+      src="https://kit.fontawesome.com/4f02d055e5.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css"/>
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+  </head>
+
+  <body>
+    <header>
+      <div class="top-bar">
+        <ul class="top-bar-ul">
+          <li><i class="fab fa-facebook-f"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li>English</li>
+          <li>Mypage</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <div class="header-bar">
+        <div class="logo">
+          <a href="./index.html"
+            ><div class="logo-climate">
+            <p>SAVE THE PLANET </p>
+          </div></a
+          >
+          <div class="closeBtn"><i class="fas fa-times fa-2x"></i></div>
```

Comment:
[OPTIONAL] On about page: The close button is still shown on the desktop version. Kindly hide it to improve your UI design.

<img width="385" alt="image" src="https://user-images.githubusercontent.com/77190203/159054118-51ff2cbe-862a-4f5e-98a5-ab78b0b04828.png">